blog/2024/04-28-jellyfin-release-10.9.0/index.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Jellyfin 10.9.0
+description: We are pleased to announce the latest stable release of Jellyfin, version 10.9.0!
+authors: joshuaboniface
+date: 2024-04-28
```

Comment:
Remove the date from metadata, it will be inferred from the folder structure.

Reply:
All the existing blog posts have this, are we sure it can be removed?

Reply:
Yes, look at the android posts